MarkupPlugin.inc.php

```diff
@@ -202,6 +217,42 @@ function getInstallSchemaFile() {
 	}
 
 	/**
+	 * @see Plugin::getActions()
+	 */
+	function getActions($request, $verb) {
+		$dispatcher = $request->getDispatcher();
+		return array_merge(
+			$this->getEnabled()?array(
+				new LinkAction(
+					'settings',
+					new RedirectAction($dispatcher->url(

```

Comment:
(I just posted this, but on the commit, not the PR -- just re-posting for completeness)

On `ojs-stable-3_0_2`, this gives me an missing class error on `RedirectAction`. Needs an import.